### ISSUE TITLE
Crash Report

### ISSUE THREAD

**Issue by MetaApo13** [Tobi823/ffupdater#343]
Stacktrace:
```
de.marmaro.krt.ffupdater.network.exceptions.NetworkException: Download of https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/mobile.v2.fenix.nightly.latest.arm64-v8a/artifacts/public/build/arm64-v8a/target.apk failed.
	at de.marmaro.krt.ffupdater.network.FileDownloader$downloadBigFileAsync$deferred$1.invokeSuspend(SourceFile:157)
	at w3.a.resumeWith(Unknown Source:8)
	at i4.l0.run(Unknown Source:102)
	at kotlinx.coroutines.internal.e.run(Unknown Source:11)
	at kotlinx.coroutines.scheduling.j.run(Unknown Source:2)
	at kotlinx.coroutines.scheduling.a$a.run(SourceFile:79)
Caused by: java.net.SocketTimeoutException: timeout
	at v4.r$c.j(Unknown Source:4)
	at v4.r$c.l(Unknown Source:8)
	at v4.r$b.read(SourceFile:132)
	at s4.c$b.read(SourceFile:18)
	at a5.r.read(SourceFile:4)
	at a5.i.read(Unknown Source:7)
	at de.marmaro.krt.ffupdater.network.ProgressResponseBody$trackTransmittedBytes$1.read(Unknown Source:5)
	at a5.r$a.read(SourceFile:4)
	at java.io.BufferedInputStream.read1(BufferedInputStream.java:286)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:347)
	at java.io.FilterInputStream.read(FilterInputStream.java:107)
	at a2.i.s(SourceFile:10)
	at de.marmaro.krt.ffupdater.network.FileDownloader.downloadBigFileInternal(Unknown Source:134)
	at de.marmaro.krt.ffupdater.network.FileDownloader.access$downloadBigFileInternal(Unknown Source:0)
	at de.marmaro.krt.ffupdater.network.FileDownloader$downloadBigFileInternal$1.invokeSuspend(Unknown Source:12)
	... 5 more
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.7.5 (131) release |
| Device | moto g52 (rhode_g, rhode, rhode) |
| Manufacturer | motorola (motorola) | 
| Supported ABIs | arm64-v8a, armeabi-v7a, armeabi |
| Android version | 12 (SDK: 31) |
| OS | ilclbld188, hudsoncm, release-keys, 1672234968000 |

**Comment by MetaApo13** [comment]
Got this issue 3-4 times, each time while updating firefox nightly

**Comment by Tobi823** [comment]
Should be fixed in 77.7.7.
